runtime/j9vm/asgct.cpp

```diff
@@ -68,8 +68,20 @@ asyncFrameIterator(J9VMThread * currentThread, J9StackWalkState * walkState)
 {
 	ASGCT_CallFrame *frame = (ASGCT_CallFrame*)walkState->userData1;
 	J9Method *method = walkState->method;
-	J9JNIMethodID *methodID = currentThread->javaVM->internalVMFunctions->getJNIMethodID(currentThread, method);
-	Assert_SC_notNull(methodID);
+	J9JavaVM * vm = currentThread->javaVM;
+	J9Class * declaringClass = J9_CLASS_FROM_METHOD(method);
+	UDATA methodIndex = getMethodIndexUnchecked(method);
+	if (UDATA_MAX == methodIndex) {
+		*(UDATA*)UDATA_MAX = UDATA_MAX;		
```

Comment:
Can we just return `J9_STACKWALK_STOP_ITERATING` instead?

Reply:
That would result in inaccurate traces being returned (which is a risk anyway I suppose). The idea was that only a completely successful stack walk would result in returning frames.

Reply:
Understood. Can we make this more clear be defining something like `#define TRIGGER_SIGSEV() *(UDATA*)UDATA_MAX = UDATA_MAX`

And adding a comment stating that `only a completely successful stack walk would result in returning frames`

Reply:
Done.